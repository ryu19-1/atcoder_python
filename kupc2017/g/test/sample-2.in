decode
100 2
49 33
35 49
