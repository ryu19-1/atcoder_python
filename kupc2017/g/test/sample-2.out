382174891210833608
